packages/framer-motion/rollup.size.config.js

```diff
@@ -0,0 +1,76 @@
+import resolve from "@rollup/plugin-node-resolve"
```

Comment:
This splits the distributed bundles from the bundles we generate for size. Makes `make build` faster when not measuring